src/test/java/org/broadinstitute/hellbender/utils/reference/ReferenceUtilsUnitTest.java

```diff
@@ -52,6 +57,34 @@ public void testLoadFastaDictionaryFromStream() throws IOException {
         }
     }
 
+    @DataProvider(name = "testData_testLoadWrongFormatFastaDictionary")
+    public Object[][] testData_testLoadWrongFormatFastaDictionary(final Method testMethod) {
+
+        Object[][] tests = {
```

Comment:
Add a comment explaining that we're trying to load a fasta file as a `dict` file to trigger a malformed file error.